### DISCUSSION TITLE
Package manager?

### DISCUSSION THREAD

**Discussion by tareksander** [shader-slang/slang#5526] [General]
In my opinion, one of the best features of modern languages compared to C and C++ is an official package manager and repository. Is something like that planned for Slang?

In the meantime, is there a list of already existing libraries in Slang?

**Comment by csyonghe** [comment]
I agree with you that an official package manager is essential for the ecosystem. We are actively considering to build such a package manager but we don't have a concrete timetable at the moment. We will share more updates once we have a clearer roadmap for it.
